public/layouts/header.php

```diff
@@ -6,6 +6,6 @@
           rel="stylesheet"
           integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB"
           crossorigin="anonymous">
-    <link rel="stylesheet" href="<?php echo url('assets/css/main.css') ?>">
+    <link rel="stylesheet" href="<?php echo url('../../assets/css/main.css') ?>">
```

Comment:
`url` helper function generates a URL stating with the `public` directory as its root path. Relative paths are not applicable to the URL structure.
While the `assets` directory is setting under the `public` directory, then the URL function can be called directly on `assets/css/main.css` to generate http://host.com/public/assets/css/main.css
```php
echo url('assets/css/main.css')
```